docs/_includes/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,353 @@
+.2+|`toc`
+.2+|Combined in a single attribute, see <<toc>>.
+
+// Ref migration guide
+|`toc-placement`
+
+// No checked
+|`options="pgwide"`
+|_not implemented_
+|DocBook attribute to make tables full page width, whatever the current indent.
+No {adr} equivalent.
+
+// Ref ap UG. I know it workls in ad tables; no ref to ad paragrphs so assume not implemented.
+|options="unbreakable"
+|
+|In {adr}, this only works for tables, not paragraphs.
+
+// http://www.methods.co.nz/asciidoc/userguide.html#X39
+// an ugly hack so not implemented
+|`plaintext`
+|_not implemented_
+|{adp} uses this to suppress inline substitutions and retain block indents when importing large blocks of plain text.
+{adr} deliberately does not implement it; the closest {adr} equivalent is a pass block.
+
+// old table
+|`replacements2`
+|`post_replacements`
+|Renamed.
+
+// does this need an explanation?
+|`presubs`
+|-
+|Not required.
+
+// rarely used so dont mention it
+//|`sgml`
+//|_not implemented_
+//|SGML is archaic and has been replaced by XML.
+
+// It may be a perversion, but it is a useful perversion!
+|`showcomments`
+|_not implemented_
+|In {adp}, turns single line comments into DocBook "remark" elements.
+{adr} considers this a bad use of comments. 
+If you want to pass remarks to the output, use an extension, or:
+----
+ifdef:showcomments[]
+<remark>Your comment here</remark>
+endif[]
+----
+
+|`specialwords`
+|_not implemented_
+|In {adp}, applies special formatting to named text.
+In {adr} this could be implemented using an extension.
+
+|`tabsize` (in-document and include directive)
+|in-document only
+|{adp} applies it to all text;
+{adr} only applies it to blocks with a verbatim content model (listing, literal, etc). Both have a default of 8. 
+For all other text, {adr} tabs are fixed at 4 spaces, see <<normalize-block-indentation>>.
+
+|====
+
+[[migrate-stylesheet]]
+=== Default HTML Stylesheet
+The {adp} and Acidoctor stylesheets are completely compatible, because their formatting is based on the same role attributes.
+If you happen to prefer the {adp} stylesheet, you can use it by copying it from the {adp} "stylesheets" directory and telling {adr} to use it with 
+
+ $ asciidoctor -a stylesheet=asciidoc.css
+
+// Worked for me, but maybe there is more to it?
+IMPORTANT: The {adr} stylesheet is designed for the web, and (unlike the {adp} stylesheet) loads some resources from the web.
+If you want to create an output for offline viewing, unset the `webfonts` attribute so that it uses system fonts instead.
+
+=== Mathematical Formulas
+
+Both {adp} and {adr} can render inline Latex and AsciiMath using `pass:[asciimath:[formula]]` or `pass:[latexmath:[formula]]` (see <<activating-stem-support>>.)
+
+In {adr} you can also set the processor separately, then use it for all inlines:
+
+----
+:stem: latexmath
+
+stem:[formula]
+----
+
+NOTE: `stem` only needs to be set once, and then only if you don't want to use the default stem processor (AsciiMath).
```

Comment:
Actually, if you want to use the STEM/math support at all, you have to define the `stem` attribute. If the value is empty it defaults to `asciimath`.